common/memtest.py

```diff
@@ -24,8 +26,16 @@ def logMemory(cls, section_name):
             MemTracker.heapy_instance = hpy()
             MemTracker.heapy_instance.setrelheap()
         
-        h = MemTracker.heapy_instance.heap()
         with open(MemTracker.log_file_path, "a") as f:
-            f.write("Memory statistics for '" + section_name + "':\n")
-            f.write(str(h) + "\n")
-            f.write("\n")
+            f.write("Memory statistics for '" + section_name + "':\n\n")
+            start = dt.datetime.now()
+            f.write("Starting TimeStamp: " + str(start))
```

Comment:
We should still have timestamps for memory stats. It's good to know how long it's taking.